templates/kubernetes/docs/contact.md

```diff
@@ -0,0 +1,48 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Contacts"
+  description: How to contact the CDK team
+keywords: help, contact, support
+tags: [contact]
+sidebar: k8smain-sidebar
+permalink: contact.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The team behind the **Charmed Distribution of Kubernetes<sup>&reg;</sup>** are
```

Comment:
I thought it was just "Charmed Kubernetes"?

Reply:
nope, definitely Charmed Distribution of kubernetes